packages/@adobe/spectrum-css-temp/components/menu/index.css

```diff
@@ -110,7 +110,8 @@ governing permissions and limitations under the License.
 .spectrum-Menu-itemLabel {
   grid-area: text;
   line-height: var(--spectrum-global-font-line-height-small);
-  word-break: break-all;
+  word-break: normal;
+  overflow-wrap: anywhere;
```

Comment:
Note: this doesn't fix the problem seen in Firefox in the ActionMenu stories where the scroll bar in the menu causes the words to wrap. If I adjust the `word-break` and `overflow-wrap` in such a way that the string don't wrap (e.g. via `word-break: normal` + `overflow-wrap: break-word`) then a single long word will NOT wrap in the menu like so: 
![image](https://user-images.githubusercontent.com/9661127/127552304-54a23432-b7aa-4542-97a0-ad703e952bb1.png)

I deemed the Firefox issue as the lesser of the two issues and thus am going ahead with the current fix